### PR TITLE
Andrew Smith IC 1

### PR DESCRIPTION
**Pull request by IcelandicGambit**
Still a little foggy but it could have been worse. Cheers

### CODE REVIEW

**Comment by IcelandicGambit** [comment]
Yeah I assumed we had an hour an a half to work on this like the first IC last inning.  My mistake.

File: lib/restaurant.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+class Restaurant
+  attr_reader :name,
+              :opening_time,
+              :dishes
+  def initialize(opening_time, name)
+    @name = name
+    @opening_time = opening_time
+    @dishes = []
+    @open_for_lunch = false
```
Comment (mikedao) [inline lib/restaurant.rb:9]: We shouldnt be storing this as an instance variable

File: lib/restaurant.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+   if @opening_time.to_i < 12
+     @open_for_lunch = true
+   else
+     @open_for_lunch
+   end
+ end
+
+ def menu_dish_names
+   @dishes.map do |dish|
+     dish.capitalize
```
Comment (mikedao) [inline lib/restaurant.rb:30]: Should be upcased, not capitalized

File: test/restaurant_test.rb [hunk 3]
```diff
@@ -51,4 +46,24 @@ def test_it_can_add_dishes
...
+  end
+
+  def test_it_can_add_menu_dish_names
+    restaurant2 = Restaurant.new('16:00', 'Il Posto')
+
+    restaurant2.add_dish('Burrata')
+    restaurant2.add_dish('Pizzetta')
+    restaurant2.add_dish('Ravioli')
+
+    restaurant2.menu_dish_names
```
Comment (mikedao) [inline test/restaurant_test.rb:67]: Test has no assertions here

**Review by mikedao — COMMENTED** [review]
Concentrate on making your tests match the spec.

Score: 2.5

**Pull request closed without merging**